tests/portal/portal_test.js

```diff
@@ -0,0 +1,64 @@
+Feature('Portal Integration with PMM');
+
+let newUser = {};
+
+AfterSuite(async ({ I }) => {
+  const users = await I.listUsers();
+  const result = users.users.filter((user) => user.email === newUser.email);
+
+  await I.deleteUser(result[0].id);
+});
+
+Scenario(
+  'PMM-T1097 Verify PMM server is connected to Portal @portal',
+  async ({
+    I, pmmSettingsPage, portalAPI, perconaPlatformPage,
+  }) => {
+    await I.Authorize();
+    pmmSettingsPage.openAdvancedSettings();
+    const publicAddress = await I.grabValueFrom(pmmSettingsPage.fields.publicAddressInput);
+
+    if (publicAddress.length !== 0) pmmSettingsPage.clearPublicAddress();
+
+    pmmSettingsPage.addPublicAddress();
+    perconaPlatformPage.openPerconaPlatform();
+    newUser = await portalAPI.getUser();
+    await portalAPI.oktaCreateUser(newUser);
+    const userToken = await portalAPI.getUserAccessToken(newUser.email, newUser.password);
+
+    await portalAPI.apiCreateOrg(userToken);
+    await perconaPlatformPage.openPerconaPlatform();
+    await perconaPlatformPage.connectToPortal(userToken, `Test Server ${Date.now()}`);
+  },
+);
+
+Scenario(
```

Comment:
Same steps are included in disconnect pmm from portal success flow so do we need this as separate test?